extension/src/state_aggregate.rs

```diff
@@ -458,14 +469,17 @@ mod tests {
             );
             let result: i64 = client
                 .select(
-                    "SELECT toolkit_experimental.duration_in('two', toolkit_experimental.state_agg(ts, state)) FROM test",
+                    "SELECT EXTRACT(epoch FROM toolkit_experimental.duration_in('two', toolkit_experimental.state_agg(ts, state))) FROM test",
                     None,
                     None,
                 )
                 .first()
                 .get_one()
                 .unwrap();
-            assert_eq!(60000000, result);
+            // XXX what have i done wrong?  it worked interactively...
+            //  left: `60`,
+            // right: `4633641066610819072
```

Comment:
@WireBaron any ideas?

Reply:
1. The problem is that `result: i64` when the return type of `EXTRACT()` is `Numeric` which is still a pointer-type. If you want this version of the code to work you need to do `EXTRACT(...)::bigint`.
2. Generally for these kinds of errors I find that text-based comparisons can make things easier to spot. If you do `EXTRACT(...)::text` it will return as a `String` and you can get the ground truth of what Postgres thinks is going on and avoid all cross-language errors.
3. Doing `duration_in(...)::text` might be better than `EXTRACT(...)::bigint` here because it will tell us the entire Interval, though I guess us since epoch does basically that 🤔 

(My first guess would be timezones, thought I don't _think_ that should be applicable here)

Reply:
Thanks!